controllers/datamovement_controller.go

```diff
@@ -296,11 +300,18 @@ func (r *DataMovementReconciler) cancel(ctx context.Context, dm *nnfv1alpha1.Nnf
 	return nil
 }
 
+func isTestEnv() bool {
```

Comment:
Two things
1) I think it's sufficient to just use the presence of the NNF_TEST_ENVIRONMENT variable as the trigger; this could then be reduced to
```
_, found := os.LookupEnv()
return found
```

2) To keep this file tidy, move this helper function to test_suite.go